sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -63,6 +63,8 @@
               .put("AVG", BeamBuiltinAggregations::createAvg)
               .put("BIT_OR", BeamBuiltinAggregations::createBitOr)
               // JIRA link:https://issues.apache.org/jira/browse/BEAM-10379
+              // .put("BIT_AND", BeamBuiltinAggregations::createBitAnd)
```

Comment:
Remove this line (BIT_AND is actually put into the table now in line 68, so the comment isn't needed).

Reply:
removed duplicate occurrence of **.put("BIT_AND", BeamBuiltinAggregations::createBitAnd)**